config/externalnamenottested.go

```diff
@@ -590,4 +590,124 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
+	"aws_ec2_transit_gateway_policy_table": config.IdentifierFromProvider,
+	// aws_ec2_transit_gateway_policy_table_association can be imported by using the EC2 Transit Gateway Policy Table identifier, an underscore, and the EC2 Transit Gateway Attachment identifier
+	"aws_ec2_transit_gateway_policy_table_association": config.TemplatedStringAsIdentifier("", "{{ .parameters.transit_gateway_policy_table_id }}_{{ .parameters.transit_gateway_attachment_id }}"),
+
+	// efs
+	//
+	// EFS Replication Configurations can be imported using the file system ID of either the source or destination file system
+	"aws_efs_replication_configuration": config.ParameterAsIdentifier("source_file_system_id"),
+
+	// emrserverless
+	//
+	// EMR Severless applications can be imported using the id
+	"aws_emrserverless_application": config.IdentifierFromProvider,
+
+	// evidently
+	//
+	// CloudWatch Evidently Feature can be imported using the feature name and name or arn of the hosting CloudWatch Evidently Project separated by a :
+	// Example: exampleFeatureName:arn:aws:evidently:us-east-1:123456789012:project/example
+	"aws_evidently_feature": config.TemplatedStringAsIdentifier("name", "{{ .external_name }}:{{ .parameters.project }}"),
+	// CloudWatch Evidently Project can be imported using the arn
+	// Example: arn:aws:evidently:us-east-1:123456789012:segment/example
+	"aws_evidently_project": config.TemplatedStringAsIdentifier("name", "arn:aws:evidently:{{ .setup.configuration.region }}:{{ .setup.client_metadata.account_id }}:segment/{{ .external_name }}"),
+	// CloudWatch Evidently Segment can be imported using the arn
+	// Example: arn:aws:evidently:us-west-2:123456789012:segment/example
+	"aws_evidently_segment": config.TemplatedStringAsIdentifier("name", "arn:aws:evidently:{{ .setup.configuration.region }}:{{ .setup.client_metadata.account_id }}:segment/{{ .external_name }}"),
+
+	// fis
+	//
+	// FIS Experiment Templates can be imported using the id
+	"aws_fis_experiment_template": config.IdentifierFromProvider,
+
+	// fsx
+	//
+	// Amazon File Cache cache can be imported using the resource id
+	"aws_fsx_file_cache": config.IdentifierFromProvider,
+
+	// glue
+	//
+	// Glue Registries can be imported using arn
+	// Example: arn:aws:glue:us-west-2:123456789012:schema/example/example
+	"aws_glue_schema": config.TemplatedStringAsIdentifier("schema_name", "{{ .parameters.registry_arn }}/{{ .external_name }}"),
+
+	// grafana
+	//
+	// No import
+	// TODO: For now API is not normalized. While testing resource we can check the actual ID and normalize the API.
+	"aws_grafana_workspace_api_key": config.IdentifierFromProvider,
+
+	// identitystore
+	//
+	// An Identity Store Group can be imported using the combination identity_store_id/group_id
+	"aws_identitystore_group": config.IdentifierFromProvider,
+	// aws_identitystore_group_membership can be imported using the identity_store_id/membership_id
+	"aws_identitystore_group_membership": config.IdentifierFromProvider,
```

Comment:
We can normalize the external-name so that it's just the (provider-assigned) `membership_id` with something like:
```suggestion
	"aws_identitystore_group_membership": config.TemplatedStringAsIdentifier("", "{{ .parameters.identity_store_id }}/{{ .external_name }}"),
```